prefs.js

```diff
@@ -31,7 +39,44 @@ function buildPrefsWidget() {
     });
     box.add(label);
     box.add(button);
+    box.add(new Gtk.Separator({}));
+    box.add(buildNichePrefsWidget());
+
     box.show_all();
 
     return box;
 }
+
+function buildNichePrefsWidget() {
+    const box = new Gtk.Box({
+        orientation: Gtk.Orientation.VERTICAL,
+        spacing: 9,
+        halign: Gtk.Align.FILL,
+        valign: Gtk.Align.CENTER,
+    });
+    box.add(new Gtk.Label({
+        label: "<b>Niche Settings</b>",
```

Comment:
I don't think `Niche Settings` is a sensible label. It doesn't mean anything with relation to what the settings will do.

Reply:
Fixed in https://github.com/pop-os/cosmic/pull/183/commits/bc95be0445eef490166697af73e3e3ce6ac094db.